docs/src/main/asciidoc/datasource.adoc

```diff
@@ -6,59 +6,57 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 = Datasources
 include::_attributes.adoc[]
 :categories: data
```

Comment:
This comment might be a bit early as I see this is the first iteration of restructuring ... but just in case I don't get a chance to say it later...

1. To ensure that this guide shows up right at the top of the doc landing page, add `getting-started` to the category list.
![image](https://user-images.githubusercontent.com/92924207/236222679-8a791876-2007-4781-bf7a-31ead7fe512a.png)

```suggestion
:categories: data,getting-started
```
Also, use the newer diataxis doc header as per the templates & [contributor guide](https://quarkus.io/guides/doc-reference#document-header).

Reply:
Great stuff! I am on it.

Reply:
Cool. I just pushed an update with these changes.

Reply:
The top of the landing page is good for this doc, since the only think we got about `datasources`, but it is not a tutorial, and only zero-config chapter could be considered as a quick-start. However, user always need to have some experience with all of this, since they are all composed as How-tos.